shopfloor/actions/stock.py

```diff
@@ -35,6 +35,21 @@ def mark_move_line_as_picked(
         )
         move_lines.picking_id.filtered(lambda p: p.user_id != user).user_id = user.id
 
+    def unmark_move_line_as_picked(self, move_lines):
+        """Reverse the change from `mark_move_line_as_picked`."""
+        move_lines.write(
+            {
+                "shopfloor_user_id": False,
+                "qty_done": 0,
+            }
+        )
+        move_lines.move_id.picking_id.write(
+            {
+                "user_id": False,
+                "printed": False,
+            }
+        )
+
```

Comment:
as it is meant to revert changes done by `mark_move_line_as_picked`, shouldn't we remove `result_package_id` as well?

Reply:
Yeah, done.